.github/workflows/dependabot-approve-merge.yml

```diff
@@ -28,14 +28,13 @@ jobs:
       pull-requests: write
 
     steps:
-      # GitHub actions bot approve
+      # Github actions bot approve
       - uses: hmarr/auto-approve-action@b40d6c9ed2fa10c9a2749eca7eb004418a705501 # v2
-        if: contains(fromJSON('["/nextcloud/vue-", "/webrtc-adapter-"]'), github.ref) != true
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
       # Nextcloud bot approve and merge request
       - uses: ahmadnassri/action-dependabot-auto-merge@45fc124d949b19b6b8bf6645b6c9d55f4f9ac61a # v2
-        if: contains(fromJSON('["/nextcloud/vue-", "/webrtc-adapter-"]'), github.ref) != true
```

Comment:
custom to Talk and we want to keep those, so webrtc and vue updates don't merge automatically